cfpb/design-system
# Added Email Form Code Snippets

Added code snippets to both e-mail signup variations.


## Additions

- Added new .less file for e-mail code snippets
- Added code snippets for both e-mail forms

## Testing

1. Visit PR preview link below
2. Visit `design-system/patterns/e-mail-signup-forms#types-1`
3. Notice new e-mail form sections
4. Click `Show Details` for code snippets

## Screenshots

![image](https://user-images.githubusercontent.com/85513449/140412923-03b77ec3-a78c-4f2c-904d-8d0d52076ffd.png)

![image](https://user-images.githubusercontent.com/85513449/140412956-988cd658-3c60-4ab0-9420-d351fc2cc698.png)

Resolves https://github.com/cfpb/design-system/issues/1010
